cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -146,3 +294,146 @@ func getDisk(workflow *daisy.Workflow, diskIndex int) *daisy.Disk {
 
 	panic("Did not find CreateDisks step.")
 }
+
+func (inflater apiInflater) traceLogs() []string {
+	return inflater.serialLogs
+}
+
+func (inflater apiInflater) addTraceLog(l string) {
+	inflater.serialLogs = append(inflater.serialLogs, l)
+}
+
+func (inflater apiInflater) inflate(ctx context.Context) (persistentDisk, error) {
+	startTime := time.Now()
+	diskName := fmt.Sprintf("shadow-disk-%v", inflater.args.ExecutionID)
+	cd := computeAlpha.Disk{
+		Name:                diskName,
+		SourceStorageObject: inflater.args.SourceFile,
+	}
```

Comment:
It will fail. Let me add a check to handle "alpha API not enabled" separately.
Currently, the plan is (confirmed with PD): 
Stage 1. land the PR so our e2e  (which uses the 'latest') can use the shadow verification; run for a couple of days so PD team collected enough data point so they can rollout to GA;
Stage 2. we change our code to call to GA API so the shadow verification can be done towards all live traffic;
Stage 3. Switch apiInflater as main. Stop shadow logic.

Reply:
At what point do we release to the general public? after stage 3?

Reply:
The shadow logic will be released to customer in stage 1.
The new API will be used in main path starting from stage 3.
BTW PD team had one more internal communication and they decided to release to beta so that they can get tested by all the customers. They will send a email to announce it.

Reply:
Based on the staging description, it looks like it's not safe to deploy `compute-image-tools` while you're in stage 1? Or is there a gating mechanism that allows you to deploy in stage 1, while only running your new code in the e2e tests?

Reply:
Hi Eric, no gate. There is no side effect calling alpha API for shadow - just got a failure. We will handle the failure properly.

Reply:
I thought shadow will use Beta API since PD team has said they will deploy to Beta yesterday? In that case, more customers will run this shadow logic as most (all?) have Beta API enabled. This is not a deal breaker, but it's something to take into account once releasing. 

I would echo Eric's comments about this change being in mainline potentially blocking our release. E.g. if we see any issues in our tests, you would need to revert these changes before we can make releases. We can avoid this by having the first stage being built from another branch, build docker images with a dedicated tag and use that tag in our e2e test (this will require a change to e2e tests).

Reply:
Stage 1 will have 2 sub steps:
1.1 Alpha, which takes effect to mostly e2e tests.
1.2 beta, which takes effect to all customers. (customers don't need to enable a beta API)

So this PR will firstly run alpha.
I have tested those images which are used in e2e and there is no problem. I don't expect to see extra functionality problems in e2e, The purpose is just to make more calls for PD team.
Plus, it's on the shadow path.
So I don't feel it's risky enough to have an additional protection. The shadow-test itself is already a protection.

Before it's been to beta, we have already ran it towards e2e for some days.